config/symbols.pspeu.stcat.txt

```diff
@@ -33,7 +32,6 @@ EntityUnkId13 = 0x09248AE8;
 EntityExplosionVariants = 0x09248D60;
 EntityGreyPuff = 0x09248ED8;
 EntityIntenseExplosion = 0x09249A08;
-MakeExplosions = 0x09249C80;
```

Comment:
This stuff will likely get conflicted by #2995 so feel free to merge that and I'll rebase this